wallet_presentation/src/commonMain/kotlin/net/thechance/mena/wallet/presentation/base/BaseViewModel.kt

```diff
@@ -66,9 +66,9 @@ abstract class BaseViewModel<STATE, EFFECT>(initialState: STATE) : ViewModel() {
         return when (throwable) {
             is NoInternetException -> ErrorState.NoInternet
             is NoDataFoundException -> ErrorState.NoDataFound
-            is UnknownException -> ErrorState.Unknown
-            is WalletException -> ErrorState.Unknown
-            else -> ErrorState.Unknown
+            is UnknownNetworkException -> ErrorState.UnknownError
+            is WalletException -> ErrorState.UnknownError
+            else -> ErrorState.UnknownError
```

Comment:
```suggestion
            else -> ErrorState.UnknownError
```

I think it's useless checks

Reply:
All of them will be mapped to the same state